ansible/roles/ocp4-workload-open-data-hub/tasks/pre_workload.yml

```diff
@@ -3,26 +3,26 @@
 
 #Workaround removing osrcp-* delete when https://github.com/openshift/openshift-restclient-python/pull/312 merged and have access to latest python restclient
 #renaming file
-- name: WORKAROUND get file name to delete
-  shell: ls /tmp/osrcp*
-  register: tempFile
-- debug:
-    var: tempFile
-- name: WORKAROUND delete cache
-  file:
-    path: "{{ tempFile.stdout }}"
-    state: absent
-  become: yes
-  register: result
-- debug:
-    var: result
+- name: WORKAROUND checking if potential /tmp/osrcp* exist to conflict with
+  block:
+  - name: WORKAROUND get file name to delete
+    shell: ls /tmp/osrcp*
```

Comment:
you can use the `find` ansible module https://docs.ansible.com/ansible/latest/modules/find_module.html